code/online_challenges/src/coderbyte/first_reverse/readme.md

```diff
@@ -0,0 +1,2 @@
+# First Reverse Program
```

Comment:
I think the README can be a bit more detailed, and the word readme should be capitalized completely.